### ISSUE TITLE
[Repo Rearchitecture] Remove github validations and rely only on catalog service validations

### ISSUE THREAD

**Issue by rudouglas** [newrelic/newrelic-quickstarts#572]
## Description 

The depreciation phase will occur at a later date. 
~ 100 days from when we finish the migration phase. 
DevEx will flip the flag when the time comes

Future change, after we are fully complete with the migration phase

## AC
 - [ ] Remove [ensure-schema-compliance](https://github.com/newrelic/newrelic-quickstarts/blob/50f2b4025682ce05e9481093347390a2f3a65991/.github/workflows/validate_packs.yml#L6)
 - [ ] Remove [ensure-quickstart-names-ids-are-unique](https://github.com/newrelic/newrelic-quickstarts/blob/50f2b4025682ce05e9481093347390a2f3a65991/.github/workflows/validate_packs.yml#L24)

### Helpful Links

Schema - https://docs.google.com/spreadsheets/d/1EmAvq4kJUUPlpvJZ-s23uKDk_KCws9VgHzG1bJtwE4Y/edit#gid=0
NerdGraph - https://api.newrelic.com/graphiql?#query=mutation%20%7B%0A%20%20nr1CatalogSubmitMetadata%0A%7D%0A
DevEx Service - https://source.datanerd.us/nr1-dev-experience/catalog-service-elixir

**Comment by zstix** [comment]
We can, potentially, remove the code to validate pull requests once we have our single source of validation truth (NerdGraph). That would mean that we don't have to wait for any duration of time to do this work.

This still needs some research to make sure that NerdGraph does the required validation that we need for our UI purposes.

An alternative is to keep both validation methods and only consider a quickstart valid if both pass (and then remove the other parts after the depreciation period).

@rudouglas to look into this further and update the acceptance criteria.

**Comment by jpvajda** [comment]
@rudouglas to confirm it seem this work is covered in this other ticket https://github.com/newrelic/newrelic-quickstarts/issues/571

**Comment by jpvajda** [comment]
The more I grok this ticket, I think it's intended to be the final step when we officially cut over to the new service, can you confirm @rudouglas

**Comment by jpvajda** [comment]
this is technically blocked as we don't want switch over yet according to @jerelmiller

**Comment by jpvajda** [comment]
moving to sustaining as it's something we need to wait on.

**Comment by jpvajda** [comment]
this is a duplicate of #668
